README.rst

```diff
@@ -8,12 +26,11 @@ In order to use this library, you first need to go through the following steps:
 
 1. `Select or create a Cloud Platform project.`_
 2. `Enable billing for your project.`_
-3. Enable the Google Cloud Trace API.
```

Comment:
Were the "enable" and "setup authentication" steps removed intentionally?

Reply:
Fixed.